build-logic/src/main/kotlin/velocity-publish.gradle.kts

```
@@ -0,0 +1,33 @@
+plugins {
+    java
+    `maven-publish`
+}
+
+extensions.configure<PublishingExtension> {
+    repositories {
+        maven {
+            credentials(PasswordCredentials::class.java)
+
+            name = "paper"
+            val base = "https://repo.papermc.io/repository/maven"
+            val releasesRepoUrl = "$base-releases/"
+            val snapshotsRepoUrl = "$base-snapshots/"
+            setUrl(if (version.toString().endsWith("SNAPSHOT")) snapshotsRepoUrl else releasesRepoUrl)
+        }
+    }
+    publications {
+        create<MavenPublication>("maven") {
+            from(components["java"])
+            pom {
+                name.set("Velocity")
+                description.set("The modern, next-generation Minecraft server proxy")
+                url.set("https://www.velocitypowered.com")
```

```suggestion
                url.set("https://papermc.io/software/velocity")
```

velocitypowered.com redirects here